notebooks/collision_avoidance/train_model.ipynb

```diff
@@ -128,16 +128,16 @@
    "source": [
     "train_loader = torch.utils.data.DataLoader(\n",
     "    train_dataset,\n",
-    "    batch_size=16,\n",
+    "    batch_size=8,\n",
     "    shuffle=True,\n",
-    "    num_workers=4\n",
+    "    num_workers=1\n",
     ")\n",
     "\n",
     "test_loader = torch.utils.data.DataLoader(\n",
     "    test_dataset,\n",
-    "    batch_size=16,\n",
+    "    batch_size=8,\n",
```

Comment:
If you need to, you might be able to drop this lower.  If 8 works for you, this is good to go though.

Reply:
Batch size 8 was found out to be optimal. 
As the batch size gets lowered from 8, the training time gradually increases, while the training time jump up with 16.